### ISSUE TITLE
Types incorrect on 4.0.0

### ISSUE THREAD

**Issue by nwalters512** [mixmaxhq/search-string#44]
See https://arethetypeswrong.github.io/?p=search-string%404.0.0.

Here's a basic reproduction: https://stackblitz.com/edit/vitejs-vite-cbevuq?file=src%2Findex.ts

Note how one needs to access the `default` property instead of accessing `parse` directly on the default export.

**Comment by nwalters512** [comment]
@raphaelbs is this library maintained? I can see about opening a PR if there's a chance it would get merged.

**Comment by bradvogel** [comment]
Yes, it is. Feel free to open a PR and we'll release a new version.

Thanks for your contribution!
